c10/util/Float8_e4m3fnuz.h

```diff
@@ -20,18 +20,10 @@
 /// the existing Float8_e4m3fn implementation.
 
 #include <c10/macros/Macros.h>
-#include <c10/util/C++17.h>
 #include <c10/util/TypeSafeSignMath.h>
 #include <c10/util/floating_point_utils.h>
 
-#if defined(__cplusplus) && (__cplusplus >= 201103L)
```

Comment:
What does removing these ifdef accomplish? Are we sure these c10s won't ever be used by OPENCL code?

Reply:
The headers they guard are unused...

Reply:
Not if compiling in C mode vs C++, right?

Reply:
Not sure whether there is a need to use in C mode, let's revert them.